src/components/virtualRepeat/virtual-repeater.js

```diff
@@ -100,6 +102,25 @@ function VirtualRepeatContainerController($$rAF, $scope, $element, $attrs) {
   /** @type {number} Amount to offset the total scroll size by. */
   this.offsetSize = parseInt(this.$attrs.mdOffsetSize, 10) || 0;
 
+  if(this.$attrs.hasOwnProperty('mdTopIndex')) {
```

Comment:
Space between if and paren (here and elsewhere, to be consistent with the rest of the file)